lib/Jsrt/JsrtDiag.cpp

```diff
@@ -129,9 +130,10 @@ CHAKRA_API JsDiagStopDebugging(
         {
             Assert(scriptContext->IsScriptContextInDebugMode());
 
-            if (FAILED(scriptContext->OnDebuggerDetached()))
+            HRESULT hr;
+            if (FAILED(hr = scriptContext->OnDebuggerDetached()))
             {
-                Debugger_AttachDetach_fatal_error(); // Inconsistent state, we can't continue from here
+                Debugger_AttachDetach_fatal_error(hr); // Inconsistent state, we can't continue from here
                 return JsErrorFatal;
```

Comment:
>return JsErrorFatal; [](start = 16, length = 20)

We won't reach here ever, can be simplified as Debugger_AttachDetach_fatal_error(scriptContext->OnDebuggerDetached())?